siliconcompiler/flows/_common.py

```diff
@@ -6,19 +6,19 @@ def setup_frontend(chip):
     frontend_flow = []
 
     if frontend in ('verilog', 'systemverilog'):
-        frontend_flow.append(('import', 'surelog'))
+        frontend_flow.append(('import', 'surelog', 'import'))
```

Comment:
This is fine, but it might be easier to read as a dict, I'm not a fan of tuples especially since indexing and changes to the format will be hard to track:
```
.append({"step": "import", 
         "tool": "surelog",
         "task": "import")

Reply:
I agree, will leave it for now. We should discuss replacements for the frontend.